ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/IcebergTableMaintainer.java

```diff
@@ -100,9 +100,13 @@ public void cleanOrphanFiles(TableRuntime tableRuntime) {
 
     // refresh
     table.refresh();
-
-    // clear dangling delete files
-    cleanDanglingDeleteFiles();
+    String version = table.properties().get("format-version");
+    if (version.equals("1")) {
+      LOG.debug("Table {} format version is v1,not need to cleanDanglingDeleteFiles", table.name());
```

Comment:
Using `clean dangling delete files` instead of `cleanDanglingDeleteFiles` for the logs to conform to English grammar.